library/packages/src/include/packages/common.rb

```diff
@@ -150,6 +150,7 @@ def RemoveAllMsg(packages, message)
     end
 
     def Install(package)
+      log.info "DEBUG: call Install(#{package.inspect})"
```

Comment:
this looks like leftover. Either use really log.debug or remove that DEBUG prefix.